docs/LibraryEvolution.rst

```diff
@@ -413,8 +420,14 @@ stored subscripts. This means that the following changes are permitted:
   accessor bodies, not the labels that are part of the subscript's full name).
 - Reordering generic requirements (but not the generic parameters themselves).
 - Adding a default argument expression to an index parameter.
+- Adding, changing, reordering, or removing property wrappers that either are 
+  implementation-detail or belong in a composition leading with an
```

Comment:
Nit: Can you maybe clarify what "leading with" means in this context, maybe with a terse example? It makes sense to me fresh from reviewing a series of property wrapper-related proposals, but it may not make sense to a general audience (or even to me in a few months!).

Reply:
Agreed, I commonly use the phrase "outermost property wrapper" to describe the first property wrapper in a composition chain

Reply:
I've changed `compositions leading with API wrappers` to `compositions where the outermost wrapper is API-level`. I've also added some brief examples of the discussed wrappers; let me know if I should change something.